OneCommunityGlobal/HighestGoodNetworkApp
# Yubo - feature- warning popup in editing media links

# Description
This is a 


## Main changes explained:
- Add a popup to if the `Media Folder` link is changed to remind and confirm such operation
- Add a **Cancel** button to withdraw the changing operation.


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user -> go to other user's profile page -> click link **Edit** under picture.
4. Try to change the `Media Folder` link and see if the popup works.
5. Click **Cancel** will withdraw the changes; Click **Confirm** will continue editing.

## Screenshots or videos of changes:
<img width="488" alt="Screenshot 2023-09-03 at 2 41 39 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/740fd145-6640-4dac-b0ec-504cc42b3ba2">

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/5b168205-d5ab-482a-a9c1-7747d72f5dc0


## Note:
You need **Admin** account to change other user's `Media Folder` link
